File: javascript/stringsdemo/StringsPlayground.js
// Declaration
let myName = "Pablo Pérez";
console.log("Hola " + myName);
// Split
let mySplittedName = myName.split(' ');
console.log("Hola " + mySplittedName[0]);
// Lower case
let sentence = "THIS SHOULD BE DISPLAYED WITH LOWER CASE";
console.log(sentence.toLowerCase());
// Upper case
let lowerSentence = "this will be displayed in upper case";
console.log(lowerSentence.toUpperCase());
// Trim
let sentenceWithSpaces = "       Test trim string        ";
console.log(sentenceWithSpaces.trim());
// Length
console.log(sentenceWithSpaces.length);

// String compare (taken from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality)
console.log('\nString comparision:');

console.log('hello' == 'hello');
// expected output: true

console.log('1' ==  1);
// expected output: true

console.log(0 == false);
// expected output: true


// Conversions
console.log(Math.PI.toFixed(4));
console.log(Math.PI.toString());
floatVal = parseFloat("321.23") + 20;
console.log(floatVal);
intVal = parseInt("321.23") + 20;
console.log(floatVal);

File: python/stringsdemo/StringsPlayground.py
# Declaration
from decimal import Decimal
from tokenize import Double


name = "Pablo Perez Martinez"
print ("Hola " + name)
# Split
splitedName = name.split(' ')
print ("Hola " + splitedName[0])
# Lower Case
sentence = 'THIS SHOULD BE CONVERTED TO LOWER CASE'
print (sentence.lower())
# upper case
lowerSentence = "this will be displayed in upper case"
print(lowerSentence.upper())
# Trim
sentenceWithSpaces = "       Test trim string        "
print(sentenceWithSpaces.strip())
# Length
print(len(sentenceWithSpaces))
# Replace
print(sentence.replace('THIS', 'ESTO'))

name2 = "Pablo Perez"
print('\nname2 == name : ')
print(name2 == name)
print("name2 != name : ")
print(name2 != name)
print("name2 > name : ")
print(name2 > name)
print("name2 <= name : ")
print(name2 <= name)

# Conversions
intNumber = 13
strVal = str(intNumber)
decimalVal = Decimal("23.231432")
convertedStr = "{:.2f}".format(decimalVal);
print("{:.2f}".format(decimalVal))